src/sentry/api/endpoints/group_tagkey_values.py

```diff
@@ -30,6 +31,15 @@ class GroupTagKeyValuesEndpoint(GroupEndpoint):
         "GET": ApiPublishStatus.PUBLIC,
     }
     owner = ApiOwner.ISSUES
+    enforce_rate_limit = True
+
+    rate_limits = {
+        "GET": {
+            RateLimitCategory.IP: RateLimit(limit=5, window=1),
+            RateLimitCategory.USER: RateLimit(limit=5, window=1),
+            RateLimitCategory.ORGANIZATION: RateLimit(limit=5, window=1),
```

Comment:
```suggestion
            RateLimitCategory.IP: RateLimit(limit=10, window=1),
            RateLimitCategory.USER: RateLimit(limit=10, window=1),
            RateLimitCategory.ORGANIZATION: RateLimit(limit=40, window=1),
```

I'd suggest we maybe start a bit more conservative and closer to our default defined in getsentry (https://github.com/getsentry/getsentry/blob/master/getsentry/conf/settings/defaults.py#L1482).